accord-core/src/main/java/accord/impl/SimpleProgressLog.java

```diff
@@ -29,6 +29,7 @@
 
 import javax.annotation.Nullable;
 
+import accord.utils.AccordConfig;
```

Comment:
I feel we should have `accord.config` package, this is likely to expand over time